src/features/studio/utils/utils.ts

```diff
@@ -0,0 +1,64 @@
+import { getTransactions } from 'src/utils/search/utils';
+import { CyberLinkSimple } from 'src/types/base';
+import { KeywordsItem } from '../studio.context';
+
+export const checkLoopLinks = async (links: CyberLinkSimple[]) => {
+  const uniqueLinks: CyberLinkSimple[] = [];
+  const loopLink: CyberLinkSimple[] = [];
+
+  for (let index = 0; index < links.length; index++) {
+    const item = links[index];
+    // eslint-disable-next-line no-await-in-loop
+    const response = await getTransactions({
```

Comment:
lets start using `soft3` and `cyber-ts` later

Reply:
need to update soft3, searchTx hasn't limit and offset, waiting for a response too long